packages/jest-util/src/__tests__/fakeTimers.test.ts

```diff
@@ -716,80 +874,90 @@ describe('FakeTimers', () => {
       const nativeSetInterval = jest.fn();
       const nativeSetTimeout = jest.fn();
 
-      const global = {
+      const global = ({
         clearInterval: nativeClearInterval,
         clearTimeout: nativeClearTimeout,
         process,
         setInterval: nativeSetInterval,
         setTimeout: nativeSetTimeout,
-      };
-      const timers = new FakeTimers({global, moduleMocker, timerConfig});
+      } as unknown) as NodeJS.Global;
+      const timers = new FakeTimers({
+        config,
+        global,
+        moduleMocker,
+        timerConfig,
+      });
       timers.useFakeTimers();
 
       // clearInterval()
       timers.runWithRealTimers(() => {
-        global.clearInterval();
+        (global as any).clearInterval();
```

Comment:
wonder why it needs to be casted to any?

Reply:
`clearInterval` (and all others) requires arguments - a callback and maybe a timeout/interval. So it's a type error to not pass them.

Reply:
we didn't flow type this test before, which is why there's such a big diff

Reply:
I've seen that, no biggie. It's good we're forced to gradually add types to tests, I'm a fan of that